pkg/proxy/integrations/mongoparser/mongoparser.go

```diff
@@ -55,8 +55,9 @@ func (m *MongoParser) ProcessOutgoing(requestBuffer []byte, clientConn, destConn
 		m.logger.Debug("the outgoing mongo in record mode")
 		encodeOutgoingMongo(requestBuffer, clientConn, destConn, m.hooks, m.logger, ctx)
 	case models.MODE_TEST:
+		logger := m.logger.With(zap.Any("Client IP Address", clientConn.RemoteAddr().String()), zap.Any("Client ConnectionID", util.GetNextID()), zap.Any("Destination ConnectionID", util.GetNextID()))
```

Comment:
Is this change required? Looks like you are adding few fields to the logger by default. So will these default fields appear in info logs too..?

Reply:
Yes these fields are required and they will only appear for error and debug logs of decodeOutgoingMongo function only. Other info logs will not be effected by this

Reply:
ok got it.